src/apps/mmfinance/mmfinance.definition.ts

```diff
@@ -0,0 +1,40 @@
+import { Register } from '~app-toolkit/decorators';
+import { appDefinition, AppDefinition } from '~app/app.definition';
+import { GroupType, AppAction, AppTag } from '~app/app.interface';
+import { Network } from '~types/network.interface';
+
+export const MMFINANCE_DEFINITION = appDefinition({
+  id: 'mmfinance',
+  name: 'MMFinance',
+  description: `MM.Finance - The largest ecosystem on Cronos with its DEX, Yield Optimizer, NFT, Algo Stablecoin & DTF`,
+  url: 'https://mm.finance/',
+  tags: [AppTag.LIQUIDITY_POOL],
+  links: {
+    twitter: 'https://twitter.com/mmfinance',
+    discord: 'https://discord.gg/mmfinance',
+    telegram: 'https://t.me/mmfinance',
+    medium: 'https://mmfinance.medium.com/',
+  },
+  groups: {
+    pool: { id: 'pool', type: GroupType.TOKEN, label: 'Pools' },
+    // farmV2: { id: 'farm-v2', type: GroupType.POSITION, label: 'Farms' },
+    // syrupStaking: { id: 'syrup-staking', type: GroupType.POSITION, label: 'Syrup Pools' },
+    // syrupMmf: { id: 'syrup-mmf', type: GroupType.POSITION, label: 'Syrup Pools' },
+    // autoMmf: { id: 'auto-mmf', type: GroupType.POSITION, label: 'Auto MMF', isHiddenFromExplore: true }, // This is the old staking mmf staking contract, and the rewards are being counted towards the TVL
+    // ifoMmf: { id: 'ifo-mmf', type: GroupType.POSITION, label: 'IFO Mmf', isHiddenFromExplore: true },
```

Comment:
With these commented out, you'll have type errors in your files that are referencing these groups. Uncomment these line, and simply have the files commented out in the module like you have already done.